### PR TITLE
[Build/Release/CI] add release docker image

### PR DESCRIPTION
**Pull request by ZhiHanZ**
I hereby agree to the terms of the CLA available at: https://datafuse.rs/policies/cla/

## Summary
Add release docker image after release binary uploaded

## Changelog

- Build/Testing/CI

## Related Issues

fixes https://github.com/datafuselabs/datafuse/issues/1151

## Test Plan

Unit Tests

Stateless Tests

### CODE REVIEW

File: .github/workflows/datafuse-release.yml [hunk 1]
```diff
@@ -67,3 +67,41 @@ jobs:
...
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
+
+      - name: Build and push
+        id: docker_build
+        uses: docker/build-push-action@v2
+        with:
+          push: true
+          tags: ${{ secrets.DOCKERHUB_NAMESPACE }}/datafuse:${{ steps.tag_name.outputs.VERSION }}
```
Comment (PsiACE) [inline .github/workflows/datafuse-release.yml:102]: may not get `steps.tag_name.outputs.VERSION` now

Comment (PsiACE) [reply .github/workflows/datafuse-release.yml:102]: IMG_20210810_142338

Comment (ZhiHanZ) [reply .github/workflows/datafuse-release.yml:102]: I think while publish job finished(https://github.com/datafuselabs/datafuse/issues/1397), docker start to build, thus this should work

Comment (ZhiHanZ) [reply .github/workflows/datafuse-release.yml:102]: strange, let me adopt some other methods

Comment (ZhiHanZ) [reply .github/workflows/datafuse-release.yml]: https://github.community/t/how-to-get-just-the-tag-name/16241/7 adopted this trick

Comment (PsiACE) [reply .github/workflows/datafuse-release.yml]: @ZhiHanZ line 41 also need fix

Comment (ZhiHanZ) [reply .github/workflows/datafuse-release.yml]: https://github.com/ZhiHanZ/hello-github-actions/runs/3288135110?check_suite_focus=true trick works, I think install.sh scripts should be changed accordingly

**Review by PsiACE — APPROVED** [review]
LGTM, Thanks

**Pull request merged**